.bazelrc

```diff
@@ -1,5 +1,7 @@
-common --action_env=BAZEL_CXXOPTS=-std=c++17
-common --cxxopt='-std=c++17'
+# gnu++17 is needed to avoid issues with openfhe support for __int128_t
+# see https://github.com/openfheorg/openfhe-development/issues/669
```

Comment:
could you please file a quick TODO in the WORKSPACE for openfhe to remove this when they update their release?